po/SConscript

```diff
@@ -91,7 +91,7 @@ if "pot-update" in COMMAND_LINE_TARGETS:
             env.Command(new_pot, source_pot, ["msgcat --sort-by-file $SOURCES -o $TARGET", Delete(source_pot)])
         env.Command(pot, new_pot, Action(update_pot))
 
-    env.Alias("pot-update", "../translations")
+    env.Alias("pot-update", "../translations", "python utils/po_stat.py --update-cfg --textdomains=wesnoth,wesnoth-editor,wesnoth-help,wesnoth-lib,wesnoth-multiplayer,wesnoth-tutorial,wesnoth-units")
```

Comment:
What exactly does this do again? It won't interfere with the normal function of the `pot-update` target, will it?

Reply:
It calculates translated% (only non-fuzzy strings) in the named domains, and adds a field "precent" to `data/languages/*.cfg`, in order for Wesnoth to read it in runtime when displaying language selection menu.

Reply:
Sorry, my question was more about the scons functionality - the meaning of the third parameter to `Alias`. Or more importantly, it will still do the normal `pot-update` as well as `po_stat`, right?

Reply:
Nothing interacting with `pot-update` artifacts here, except for, if this step fails (unlikely - only if internal error happens), `pot-update` will fail.

Reply:
Third argument is an extra action that will be attached to the alias and will be executed AFTER building all things that depend on pot-update.